std/functional.d

```diff
@@ -702,6 +694,14 @@ template partial(alias fun, alias arg)
     }
 }
 
+///
+unittest
+{
+    int fun(int a, int b) { return a + b; }
+    alias partial!(fun, 5) fun5;
```

Comment:
oh didn't know that - just copied the old one.
Thanks for the info - updated ;-)